jobs/periodic/match-follow-up/index.ts

```diff
@@ -4,6 +4,7 @@ import mailjet from "../../../common/mails/mailjet";
 import { Match } from "../../../common/entity/Match";
 import { sendMatchFollowUpStudent, sendMatchFollowUpPupil } from "../../../common/mails/match-follow-up";
 import * as moment from "moment-timezone";
+import * as Notification from "../../../common/notification";
```

Comment:
These match follow up mails (and actually all other follow up mails) should be scheduled upon specific actions (as it is already implemented by you). Hence these retrieving functions will be obsolete once we switched totally to this notification system. So there shouldn't be anything to do for you here. 😉

Reply:
Indeed, not sure how that import ended up there 😅